custom_components/grocy/json_encode.py

```diff
@@ -17,6 +18,8 @@ def default(self, o: Any) -> Any:
             return o.barcode
         if isinstance(o, datetime):
             return o.isoformat()
+        if isinstance(o, date):
```

Comment:
This should not be separate a if statement.
The second argument to `isinstance` supports tuple, so it should be added to the statement above it.

The issue reported in #185 was resolved with #175 which is a part of [v4.1.4b](https://github.com/custom-components/grocy/releases/tag/v4.1.4b)